src/com/nutrons/nu17/RobotMap.java

```diff
@@ -24,8 +24,9 @@
 	public static final int GYRO = 0;
```

Comment:
We know the RobotMap class is full of port numbers, eliminate that from the comments throughout the class